### DISCUSSION TITLE
Learncard and Zapier

### DISCUSSION THREAD

**Discussion by iancrogers** [learningeconomy/LearnCard#199] [Help]
We're using the Xperiencify LMS for our courses. They use Zapier to connect with external services.

We want to send graduation data to LearnCard; would it be possible to do this through Zapier?

Thank you
Ian Rogers
Metaverse Masterclass

**Comment by Custard7** [comment]
@iancrogers yes! Definitely is possible. 

If you already have zapier setup, you could send the graduation data to a post request to a LearnBridge endpoint.

I'll be back in the office on Monday and can describe this in more detail!

Are you able to more fully describe what your goal is?

**Reply by iancrogers** [reply]
Hi @Custard7, thanks for the response.  In very basic terms, we want to automatically issue badges to learners after they complete a course. Xperiencify uses Zapier to send graduation and user data.

We are new to LearnCard, and have only played around a bit with the Discord bot, so any direction you can provide is most appreciated.
